Tribler/community/tunnel/tunnel_community.py

```diff
@@ -123,10 +123,21 @@ def enabled(self):
     def sendto(self, data, destination):
         if self.check_num_packets(destination, False):
             if TunnelConversion.is_allowed(data):
+                ip_address = destination[0]
```

Comment:
Better to check if it's an actual IP address than to capture all exceptions and then try to resolve the name. (maybe using ```dispersy.util.is_valid_address()```)